master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: Accessing Calico policy with Calico as a network plugin
+title: Using Calico with Docker
```

Comment:
This title doesn't match how we're referring to it elsewhere in the docs, which I think is pretty confusing.

Simply fix is just to make the title "Advanced Policy".